test/validation/api/time/time.c

```diff
@@ -975,6 +977,136 @@ static void time_test_global_sync_control(void)
 	time_test_global_sync(1);
 }
 
+static int run_time_global_thread(void *arg)
+{
+	global_shared_mem_t *gbl = arg;
+	const odp_queue_t queue = gbl->queue;
```

Comment:
In the to-be-removed example app the test uses a separate queue for each worker, ensuring that the events cannot be processed by single worker thread. With this code it could happen that only one worker thread manages to dequeue anything, which would reduce the inter-thread time stampt test to the checks between the single worker and the main thread at termination. Perhaps you could consider having separate queues for the workers and e.g. have the events traverse the workers in a circular fashion (or even randomized as in the to-be-removed app).

Reply:
Implemented in V2.